content/feed/v2.md

```diff
@@ -18,19 +18,60 @@ If you build your feed's xml carefully (simply copy the first 5 lines from our e
 
 **If your feed is invalid with respect to our XML schema all contained data will be ignored! So keep your feed valid under all circumstances!**
 
-## Changes to [v1](/feed/v1/)
+## *openmensa* root element
 
-* new *openmensa* root element
-* *name*s need to have less than 250 chars
-* rename cafeteria to *canteen*
-* price *role*s
-* closing times on day granularity
+The root element is `openmensa`. It encapsulate all other attributes. This element will be preserved across future feed versions. The *version* attribute must equal 2.0.
```

Comment:
As the version can be alternativly specified by a HTTP header I'd drop the "attribute" here to clarify that only the version must equal "2.0.": `The *version* must equal 2.0.`